src/Splitio/Domain/RedisConfigurationValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Splitio.Services.Client.Classes;
+using Splitio.Services.Logger;
+using Splitio.Services.Shared.Classes;
+using System;
+
+namespace Splitio.Domain
+{
+    public static class RedisConfigurationValidator
+    {
+        private static readonly ISplitLogger _log = WrapperAdapter.Instance().GetLogger(typeof(RedisConfigurationValidator));
+
+        public static void ValidateRedisOptions(ConfigurationOptions options)
```

Comment:
what happen if `CacheAdapterConfig` is null ?

Reply:
We do check that in Factory before calling validate